lib/components/viewers/stop-viewer.js

```diff
@@ -368,7 +367,17 @@ class StopViewer extends Component {
           >
             <Icon Icon={Search} style={{ marginLeft: '0.2em' }} />
           </button>
-          {!isFlex && (
+          <button
+            className="link-button pull-right"
+            onClick={this._viewNearby}
+            style={{ fontSize: 'small' }}
+          >
+            {/* FIXME: What icon should we use? */}
```

Comment:
Any thoughts for https://fontawesome.com/icons/magnifying-glass-location?f=classic&s=solid?